src/hotspot/share/nmt/vmatree.cpp

```diff
@@ -29,10 +29,74 @@
 
 const VMATree::RegionData VMATree::empty_regiondata{NativeCallStackStorage::StackIndex{}, mtNone};
 
-const char* VMATree::statetype_strings[3] = {
-  "reserved", "committed", "released",
+const char* VMATree::statetype_strings[4] = {
+  "released","reserved", "'only-committed", "committed",
```

Comment:
Extra quote in `"only-committed"`. Why is this new state required by the way?

Reply:
Removed the extra quote.
Do you remember the new bit flags for state? 
0 = Released, 1 = Reserved, 2= Committed(but not Reserved),
 3= both Reserved and Committed